src/Graph.js

```diff
@@ -609,9 +609,12 @@ const mkConsumptionLeisure = function(board, options) {
 
 class ConsumptionSavingGraph extends Graph {
     make() {
+        const me = this;
+
         const f1 = function(x) {
             // c2 = y2 + (1 + r)(y1 + omega - c1)
-            return 0 + (1 + 1) * (0 + 1 - x);
+            return me.options.gA2 + (1 + me.options.gA4) *
```

Comment:
I like the abstraction you've achieved by declaring a set of generic/configurable variables for the various graph types. One note: the generic variable names might be confusing going forward as they don't convey implicit meaning. Perhaps some doc for each graph type that maps, say gA2 to omega or whatever?

Reply:
Yeah, that's definitely due. I'll open a new PR to make a more formal note for what I'm using each of these values for in the different graph types of Graph.js.